spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -35,7 +35,7 @@ def handle_two_step(response)
         result = choose(*available)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
-      elsif r.body.kind_of?(Hash) && r.body["phoneNumberVerification"].kind_of?(Hash)
+      elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Hash)
```

Comment:
`trustedPhoneNumbers` is an array, so this check will fail. In my test code `r.body["trustedPhoneNumbers"][0].kind_of?(Hash)` worked so an extra check on the array and the first element may be nice (soon all elements? not sure how Apple will change this)